frontend/packages/console-app/src/components/nodes/node-dashboard/queries.ts

```diff
@@ -35,11 +35,13 @@ const queries = {
     `node_memory_MemTotal_bytes{instance='<%= node %>'} - node_memory_MemAvailable_bytes{instance='<%= node %>'}`,
   ),
   [NodeQueries.MEMORY_TOTAL]: _.template(`node_memory_MemTotal_bytes{instance='<%= node %>'}`),
-  [NodeQueries.POD_COUNT]: _.template(`kubelet_running_pod_count{instance=~'<%= ipAddress %>:.*'}`),
```

Comment:
for consistency i switched this to using the node name, which meant getUtilizationQueries no longer had a reason to take in the ipAddress of the node